AP_article_builder.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+from bs4 import BeautifulSoup
+import pprint
+import csv
+import datetime
+import nltk
+import requests
+import csv
+import json
+import spacy
+from spacy import displacy
+import pprint
```

Comment:
Please remove all unused imports. Do the same for all files.

Reply:
Resolved in my local repo.